### ISSUE TITLE
Sticky plugin makes div larger

### ISSUE THREAD

**Issue by iammichiel** [zurb/foundation-sites#8427]
Hi, 

I am having a strange issue when using the sticky plugin. 
I simply create two divs `large-9` and `large-3`. The `large-3` is a sticky div for `large-9`.

``` html
<div class="row">
    <div id="leftbar" class="large-9 columns">
    </div>

    <div id="rightbar" class="columns large-3" data-sticky-container>
        <div class="sticky" data-sticky data-anchor="leftbar">

        </div>
    </div>
</div>
```

Without the sticky part, the `#rightbar` is 369 pixels wide. When adding the sticky class, it gets 12 pixels larger.

**Comment by gehasia** [comment]
Please provide **full** html+css code to reproduce the bug.
You tell rightbar is 369px wide, but the width will depend of your browser windows width, do you mean you fixed the column width ???

**Comment by andycochran** [comment]
If you can, please reproduce this in a [CodePen](http://codepen.io/rafibomb/pen/ZQEGog).

**Comment by iammichiel** [comment]
Yep, sure. Here is a link to a [Codepen](http://codepen.io/iammichiel/pen/WwmqmZ) illustrating the problem.

I can now go a little further in my explanation, when adding the .sticky behaviour, the right gutter is no longer displayed.

**Comment by andycochran** [comment]
@iammichiel, thanks for clarifying. 

I actually just noticed this bug in the docs today. The sticky space images spill into the right gutter. This isn't visually evident because there's some bad markup. There are columns nested in a column without a row: 

```html
<!-- ```html_example -->
<div class="row">
  <div class="columns small-12">
    <div class="columns small-6" id="example1" data-something>...</div>
    <div class="columns small-6 right" data-sticky-container>...
```
The problem is very evident if you remove that superfluous `<div class="columns small-12">`.

**Comment by andycochran** [comment]
@gehasia, you're good with .js and familiar with Sticky, right? Would you like to take a stab at fixing this? I'd love to see a PR.

**Comment by gehasia** [comment]
Sure, i'll try to have a look on it next week probably (as i'll be busy for few days) !

**Comment by Jflem1290** [comment]
@gehasia Hey! I was wondering if you had time to take a look at this issue. I would love to collaborate and get this issue resolved for the community.

**Comment by gehasia** [comment]
@Jflem1290 
Hi ! Unfortunately i don't have any time right now, if you want to look at it, feel free to do it !

**Comment by Jflem1290** [comment]
@gehasia Sound great!

**Comment by Jflem1290** [comment]
@gehasia @andycochran @iammichiel 
I think I may have pinpointed the issue in the` foundation.sticky.js.`

```
 var _this = this,
   newElemWidth = this.$container[0].getBoundingClientRect().width,
   comp = window.getComputedStyle(this.$container[0]),
   pdng = parseInt(comp['padding-right'], 10);

 this.$element.css({
   'max-width': `${newElemWidth - pdng}px`
 });

```

The calculation between `newElemWidth` and `pdng` seems to be insufficient.

To break it down (in this case): 
     `newElemWidth = 300px`
     `pdng = 15px`
     `newElemWidth - pdng = 285px`

The max width of the container should be 270px not 285.

So, if we take the `pdng` and multiply it by 2, we will get 30. Therefore, 300-30=270.
For example:
`pdng = parseInt(comp['padding-right'], 10) * 2; = 30` 

I've also received the same output when using a regex of 25 instead of the default 10.
``pdng = parseInt(comp['padding-right'], 25) = 30`


I am not sure if this is just for this case but it would be great to get it tested in other cases. Feel free to download the `foundation.sticky.js` test file to your project and test. 
[foundation.sticky.js.zip](https://github.com/zurb/foundation-sites/files/402156/foundation.sticky.js.zip)

**Comment by kball** [comment]
I believe this will be addressed by https://github.com/zurb/foundation-sites/pull/9298

**Comment by kball** [comment]
#9298 was just merged - this should be fixed in 6.3
